layers/+completion/unite/packages.vim

```diff
@@ -3,6 +3,6 @@
 
 MP 'Shougo/unite.vim'
 
-MP 'ctrlpvim/ctrlp.vim',      { 'on': ['CtrlP', 'CtrlPMRU'] }
+MP 'ctrlpvim/ctrlp.vim'
 
-MP 'FelikZ/ctrlp-py-matcher', { 'on': ['CtrlP', 'CtrlPMRU'] }
+MP 'FelikZ/ctrlp-py-matcher'
```

Comment:
For these two ctrlp plugins, if the user doesn't use tmux layer,  remain the on-demand loading, keep them as they were.

```vim
if core_config#LayerLoaded('tmux')
   MP 'ctrlpvim/ctrlp.vim',      { 'on': ['CtrlP', 'CtrlPMRU'] }
   MP 'FelikZ/ctrlp-py-matcher', { 'on': ['CtrlP', 'CtrlPMRU'] }
else
   MP 'ctrlpvim/ctrlp.vim'
   MP 'FelikZ/ctrlp-py-matcher'
endif
```